portugues/amigo/incomodar/aplicativo.md

```diff
@@ -0,0 +1 @@
+Você fica triste e desiste do aplicativo.
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.